mysql-test/suite/innodb/t/xtradb_compressed_columns_alter_table.test

```diff
@@ -85,3 +85,25 @@ eval ALTER TABLE $original_table_name DISCARD TABLESPACE;
 # cleaning created compression dictionaries
 --eval DROP COMPRESSION_DICTIONARY $dict1_name
 --eval DROP COMPRESSION_DICTIONARY $dict2_name
+
+--echo #
+--echo # Bug PS-1108 : Changing a column from uncompressed to compressed for JSON crashes the server
+--echo #
+
+CREATE TABLE t1 (json_variables JSON);
+INSERT INTO t1 VALUES('{"name":"json test"}');
+ALTER TABLE t1 MODIFY json_variables JSON COLUMN_FORMAT COMPRESSED;
```

Comment:
One more doubt. Can we also ALTER from existing JSON column compressed to JSON compressed with dictionary? See if this case is covered. if not, good to add here.

Reply:
@nitendra-bhosle please check this scenario as well

Reply:
@satya-bodapati : I have update test case with this scenario, Please review.